plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/rating-filter/block.json

```diff
@@ -28,6 +28,10 @@
 			"type": "boolean",
 			"default": false
 		},
+		"minRating": {
+			"type": "string",
```

Comment:
Why not integer/number?

Reply:
It will used straight in the `<RadioControl />` options item, where it expects a string.
We can cast it like a number, tho.

Reply:
Not sure why my comment here is missing. The original one is:

> I see, let's keep using string then. Thanks for your explanation!